notification-channel-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_channel.proto

```diff
@@ -13,8 +13,22 @@ message NotificationChannelMutableData {
   string channel_name = 1;
   repeated EmailChannelConfig email_channel_config = 2;
   repeated WebhookChannelConfig webhook_channel_config = 3;
+  repeated AwsS3BucketChannelConfig s3_bucket_channel_config = 4;
```

Comment:
the proto looks good to me, let's get the validation in and then we can merge.

Reply:
>I'd also ask - any reason not to? The validation code we're discussing is probably a few lines, no more than 10. It'd take less time than it took me to write this comment.

Right, no particular reason of not doing this in this PR. Just wanted to know the logic behind this.